src/utils.ts

```diff
@@ -358,7 +358,10 @@ function validateTutorialNameInput(value: string, tutorialsForValidation : strin
 		if (value.trim().length === 0) {
 			return "Empty tutorial name is not allowed";
 		}
-		if (tutorialsForValidation && tutorialsForValidation.indexOf(value) > -1) {
+		if (value.endsWith(' ')) {
```

Comment:
have you tried when starting with space? not tried but I think that it will needed to check for it too

Reply:
It works. The additional trim in line 364 covers it. (Just tried locally)

Reply:
I can add an additional validation check for it to be more explicit, but the trim resolves it even if the space is there (it just doesn't complain about it)

Reply:
added